LICENSE

```diff
@@ -118,4 +118,36 @@ For more information, please see
 ----------------------------------
 
 NOTE: ace and jsoneditor in the regolith/static/ dir are covered under their own 
-licences.
\ No newline at end of file
+licences.
+
+For conftest.py
+
+BSD 3-Clause License
+
+Copyright (c) 2017, Anthony Scopatz
+All rights reserved.
```

Comment:
Simply add this statement to the conftest.py, you don't need to modify the license file.

Reply:
I just need those two lines right?